src/components/event-widget/event-widget.tsx

```diff
@@ -19,6 +19,20 @@ interface EventWidgetProps {
     wrapperStyles?: ThemeUICSSObject;
 }
 
+const formatDate = (d: string) => {
```

Comment:
don't we have this function in @max-melo code already.

Reply:
i have a function for displaying a range of dates, but it can be used to format individual dates if the 2nd date in the range is passed as undefined. 

it may be worth having a util function that formats one date specifically to make the code less confusing, but i think it might be worth moving this to `src/utils/format-date.ts`

Reply:
agree with @max-melo on this. it might be worth moving this to src/utils/format-date.ts

Reply:
sounds good, I tried passing a single date to Max's `formatDateRange`, but it had some formatting I don't need on the Industry Details page, so I'm just going to include this function in `utils/format-date.ts` in fear of breaking his functionality elsewhere by changing it.